core/src/main/java/org/geysermc/geyser/translator/protocol/java/JavaLoginTranslator.java

```diff
@@ -109,9 +109,11 @@ public void translate(GeyserSession session, ClientboundLoginPacket packet) {
 
         session.sendDownstreamPacket(new ServerboundCustomPayloadPacket("minecraft:brand", PluginMessageUtils.getGeyserBrandData()));
 
-        // register the plugin messaging channels used in Floodgate
+        // Register the plugin messaging channels used in Floodgate and emotes
         if (session.getRemoteAuthType() == AuthType.FLOODGATE) {
             session.sendDownstreamPacket(new ServerboundCustomPayloadPacket("minecraft:register", PluginMessageUtils.getFloodgateRegisterData()));
+        } else {
+            session.sendDownstreamPacket(new ServerboundCustomPayloadPacket("minecraft:register", PluginMessageUtils.getEmoteRegisterData()));
```

Comment:
Why is the emote channel not registered when using Floodgate?

Reply:
It is; see the PluginMessageUtils changes here.